components/sidebar/sidebar_channel/index.js

```diff
@@ -77,16 +78,24 @@ function makeMapStateToProps() {
         let channelTeammateDeletedAt = 0;
         let channelTeammateUsername = '';
         let channelTeammateIsBot = false;
+        let botLastIconUpdate = 0;
         let channelDisplayName = channel.display_name;
+        let botIconUrl = null;
         if (channel.type === Constants.DM_CHANNEL) {
             teammate = getUser(state, channel.teammate_id);
             if (teammate) {
                 channelTeammateId = teammate.id;
                 channelTeammateDeletedAt = teammate.delete_at;
                 channelTeammateUsername = teammate.username;
                 channelTeammateIsBot = teammate.is_bot;
+                botLastIconUpdate = teammate.bot_last_icon_update;
+            }
+            if (channelTeammateIsBot) {
+                if (typeof botLastIconUpdate !== 'undefined' &&
```

Comment:
I think this check can just be `if (botLastIconUpdate !== 0) {`

Reply:
yes. Changed that done a quick test. All updated now.